impeller/renderer/backend/vulkan/render_pass_vk.cc

```diff
@@ -433,4 +442,13 @@ bool RenderPassVK::TransitionImageLayout(uint32_t frame_num,
   return true;
 }
 
+const ContextVK& RenderPassVK::GetContextVK() const {
+  if (auto context = context_.lock()) {
+    const auto& context_vk = ContextVK::Cast(*context);
+    return context_vk;
+  }
+
+  FML_UNREACHABLE();
```

Comment:
Maybe just a `VALIDATION_LOG` instead?

This seems like it could happen if the process is shutting down, for example, and end up causing a crash that worries developers.